Makefile

```diff
@@ -0,0 +1,32 @@
+all: help
+
+help:
+	@echo "This is a short hand for running popular tasks."
+	@echo "Please check the documentation on how to get started"
+	@echo "or how to set-up the different environments."
+
+build-all: build-riscv-compliance build-simple-system build-arty
+
+# RISC-V compliance
+build-riscv-compliance:
+	fusesoc --cores-root=. run --target=sim --setup --build lowrisc:ibex:ibex_riscv_compliance --RV32M=1 --RV32E=0
+
+# Simple system
+build-simple-system:
+	fusesoc --cores-root=. run --target=sim --setup --build lowrisc:ibex:ibex_simple_system --RV32M=1 --RV32E=0
+sw-simple-hello:
+	make -C examples/sw/simple_system/hello_test
+run-simple-system: sw-simple-hello
+	build/lowrisc_ibex_ibex_simple_system_0/sim-verilator/Vibex_simple_system --raminit=examples/sw/simple_system/hello_test/hello_test.vmem
```

Comment:
Please depend on the output, not on the step.

```makefile
sw-simple-hello: examples/sw/simple_system/hello_test/hello_test.vmem

examples/sw/simple_system/hello_test/hello_test.vmem:
	make -C examples/sw/simple_system/hello_test

run-simple-system: examples/sw/simple_system/hello_test/hello_test.vmem
	build/lowrisc_ibex_ibex_simple_system_0/sim-verilator/Vibex_simple_system --raminit=examples/sw/simple_system/hello_test/hello_test.vmem
```

Not adding a dependency to the `Vibex_simple_system` is fine here to avoid needless rebuilds of the simulator (at least until fusesoc can do no-op builds). But you might want to add a test to the run-simple-system target to check if the `Vibex_simple_system` file exists, and error out with a help message ("run build-simple-system") otherwise.